zanata-war/src/test/java/org/zanata/util/ServiceLocatorTest.java

```diff
@@ -50,6 +55,23 @@
         SLF4JBridgeHandler.install();
     }
 
+    @Parameters(name = "{index}: fib[{0}]={1}")
+    public static Iterable<Object[]> data() {
+        return Arrays.asList(new Object[][] { { 0, 0 }, { 1, 1 } });
+    }
+
+    @Parameter(0)
+    private int fInput;
+
+    @Parameter(1)
+    private int fExpected;
+
+
+    @Test
+    public void testFib() {
+        assertEquals(fExpected, (fInput));
+    }
+
```

Comment:
Both tests are really just examples to demonstrate Parameters, to serve
until we get around to porting a DataProviderRunner test.

On 4 March 2016 at 15:47, Carlos Munoz <notifications@github.com> wrote:

> In zanata-war/src/test/java/org/zanata/util/ServiceLocatorTest.java
> <https://github.com/zanata/zanata-server/pull/1106#discussion_r54992478>:
>
> > +    public static Iterable<Object[]> data() {
> > +        return Arrays.asList(new Object[][] { { 0, 0 }, { 1, 1 } });
> > +    }
> > +
> > +    @Parameter(0)
> > +    private int fInput;
> > +
> > +    @Parameter(1)
> > +    private int fExpected;
> > +
> > +
> > +    @Test
> > +    public void testFib() {
> > +        assertEquals(fExpected, (fInput));
> > +    }
> > +
>
> I think this whole block is repeated from the FibonacciTest above.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/1106/files#r54992478>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat